src/azure-cli/azure/cli/command_modules/sql/tests/latest/recordings/test_sql_mi_tdebyok.yaml

```diff
@@ -13956,7 +13956,7 @@ interactions:
       accept-language:
       - en-US
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.KeyVault/vaults/sqltdebyok000005?api-version=2018-02-14
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.KeyVault/vaults/sqltdebyok000005?api-version=2019-09-01
```

Comment:
You just replace the api version string and no actual live test running for these scenarios, right? Is it any problem when running test in live mode?

Reply:
I didn't. I am blocked by `azdev` indefinitely hang on Linux. Still working on it.